### DISCUSSION TITLE
2022/10/31/articles-about-airway-beacons-and-arrows

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#1] [Announcements]
# 2022/10/31/articles-about-airway-beacons-and-arrows

http://localhost:4000/2022/10/31/articles-about-airway-beacons-and-arrows

**Comment by jengalas** [comment]
Posted by Christian Ackerley
20 November 2022 at 11:13 am

_[Imported from WordPress]_

Zhana –

Thanks again for all your good work documenting the history of airmail/airway beacons. I had a good laugh when I checked the GPS coordinates of the tower site located in Iowa, just across the Missouri River from Blair, Nebraska. I’m about 95% certain that site was the original location of my on-course light. What made me laugh so hard is how easy it was to find the site on your map! Do you have any idea how many hours I spent pouring over USGS topo maps, the original 1927 map of the Sioux Falls-Omaha Airmail Route, and and Google Maps’ satellite view tool before I located that site? Of course, that was a number of years ago, long before good people such as yourself took on the task of locating these sites. Anyway, I have two questions. The federal government maintained many of the Airway Beacon sites until the early 1970’s. By then, those sites had rotating beacons, only. Do you know when the government modernized the Airway Beacon sites ny eliminating the on-course lights? I know that electronic navigation aides came into widespread use some time in the 1950’s. That eliminated the need for the on-course lights. I would really like to know when the on-course lights actually were removed from the Airway Beacon towers. My second question is, did any agency or organization take over maintenance of the Airway Beacon sites in Montana after those oh-so-far-sighted public “servants” in the Montana legislature removed that line from the Montana Department of Transportation budget? If anyone knows the answer to this question, please respond! One last thing. I believe the GPS coordinates you have posted for the tower site located across the Missouri River, East of Blair, Nebraska are ever so slightly off. Pull up those GPS coordinates on Google Maps. Around 75 yards South by Southwest of the “pin” you will see a square foundation in the middle of a cornfield that looks an awful lot like an Airway Beacon tower foundation. What do you think? Thanks again! Chris Ackerley, Hope, Arizona.
